ShadowDOM/explainer.md

```diff
@@ -325,6 +325,28 @@ CSS Modules are not the only type of module - there are also JavasScript, JSON,
 | WASM           | `import {foo} from "bar.wasm" with {type: "wasm"};`      | `<script type="wasm-module" specifier="/bar.wasm"></script>`              |
 
```

Comment:
One of the drawbacks of this approach was the syntax. I think we had discussed an alternative that would feel more natural, although the exact details are escaping me. Do we want to update the syntax or add a section for an alternative syntax as part of this in case it helps when we go to re-review with the TAG?

Reply:
The tricky thing is that we can't really change the syntax of Declarative CSS Modules without diverging from the Imperative version significantly. Let's leave this open until @dandclark returns - he might have some ideas here.

Reply:
Per discussion earlier, I converted all to `<style>` from `<script>` and simplified the `specifier` values to look less URL-like